tests/test_adapter.py

```diff
@@ -63,7 +63,7 @@ def test_substitute_missing_expansion(self):
 class TestGenie3JobSubmission(unittest.TestCase):
     """Test Genie 3 job submission."""
 
-    @patch('pygenie.utils.requests.request')
+    @patch('pygenie.utils.requests.sessions.Session.request')
```

Comment:
This seems like a weird patch before and after.  Why are we patching the requests imported in utils.  Wouldn't we expect to just patch requests directly?

Reply:
I imagined it was to work around some import issues, but just patching requests worked. Updated